wasm/Cargo.toml

```diff
@@ -4,12 +4,17 @@ description = "Test env for WASM"
 version = "1.0.0"
 edition = "2021"
 
+[features]
+default = []
+wasm_opt = ["plonky2_field/wasm_opt"]
```

Comment:
when compiling this crate, enable the new mul implementation with `features = wasm_opt`